backend/src/Repository/StoreOwnerProfileEntityRepository.php

```diff
@@ -130,4 +130,16 @@ public function getStoreOwnerBranchesByStoreOwnerProfileIdForAdmin(int $storeOwn
             ->getQuery()
             ->getResult();
     }
+
+    public function getStoreOwnersProfilesCountByStatusForAdmin(string $storeOwnerProfileStatus): string
```

Comment:
please make sure the return result is a string, and try it in case of null

Reply:
I tested it when it is null then it will return "0".